projects/addon-doc/components/example/example.template.html

```diff
@@ -96,13 +93,13 @@
                 [style.display]="activeItemIndex === index && index === defaultTabIndex ? 'block' : 'none'"
             >
                 <ng-content />
-                <ng-container *polymorpheusOutlet="lazyComponent$ | async as text">
+                <ng-container *polymorpheusOutlet="lazyComponent() as text">
                     {{ text }}
                 </ng-container>
             </div>
 
             <tui-doc-code
-                *tuiLet="files?.[tabs?.[index] || 0] || '' as code"
+                *tuiLet="processor()[tabs?.[index] || 0] || '' as code"
```

Comment:
Do we actually need it ?

```suggestion
                *tuiLet="processor()[tabs[index] || 0] || '' as code"
```

Reply:
fixed

Reply:
fixed